Rajitha/recursion-practice.js

```diff
@@ -0,0 +1,60 @@
+/****************************************************
+* Create a function to get the factoral of a number.
+* Input: factoral(5)  Output: 120
+****************************************************/
+let fact = 1;
```

Comment:
don't use global scope to store variables. 

wrap this function in another function. - RY

Reply:
Agreed, the point of recursion is to have a self contained function that will loop itself and spit out the result. With a global variable, it is easier to just use a loop. Try doing these problems without a global variable. Instead have a variable inside the function itself that will contain the result.

Remember how I taught you guys 2-3 weeks ago where each time the function is called again, return the result to the previous function call?